.github/labeler.yml

```diff
@@ -0,0 +1,5 @@
+kind/ui:
+  - web_src/**/*
```

Comment:
You should also include the templates

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/26962/commits/570888975e16695c078a127d66fcdcaff51d33be. This config is far from complete, but I would first like to get it running in a basic state.